{"uuid": "cd71461b-3f33-4cbb-9458-e486ad38e273", "children": ["7aa7f3f3-1e27-4b5d-b494-29c8c1ed4db3"], "befores": [{"name": "setup", "status": "broken", "statusDetails": {"message": "TypeError: 'module' object is not callable\n", "trace": "  File \"C:\\Users\\makde\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\pluggy\\_callers.py\", line 80, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\makde\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\makde\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\makde\\Desktop\\SW Testing\\Pytest framework\\CredKart_pytest_project 31-07-2023\\CredKart_Pytest_Project - Copy\\testCases\\conftest.py\", line 27, in setup\n    driver = webdriver.edge()\n             ^^^^^^^^^^^^^^^^\n"}, "start": 1691475573798, "stop": 1691475573798}], "start": 1691475573798, "stop": 1691475573802}